packages/itwinui-css/src/input/searchbox.scss

```
@@ -0,0 +1,33 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@mixin iui-expandable-searchbox {
+  width: var(--_iui-input-min-height);
+  transition: width var(--iui-duration-3) ease;
```

In response to https://github.com/iTwin/iTwinUI/pull/863#issuecomment-1535067325, how about this?
```suggestion
  transition: width var(--iui-duration-1) ease-in-out;
```

---

This is serviceable for now, but in the future I would like to use a custom cubic-bezier function. The default ones feel too bland

---

In Toasts we are using `cubic-bezier(0.4, 0, 1, 1)` but that might be too long for quick animations.

---

Added ease-in-out for now.